### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by SimonGideon**
In this Pull Request, the following accessibility checks were observed:
1. Page titles. (No issue found).
2. Image text alternatives - (Refactor alternative image texts into meaningful ones).
3. Text headings. (No issue found).
4. Color contrast (No issue found).
5. Resize - (No issue found).

### CODE REVIEW

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @SimonGideon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] THe design is cool :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -52,16 +52,16 @@ <h1>
         </div>
         <ul class="social-icons" id="social-icons">
           <li>
-            <a href="https://twitter.com/Simo_Giddy"><img src="./images/twiter.png" alt="twiter"></a>
+            <a href="https://twitter.com/Simo_Giddy"><img src="./images/twiter.png" alt="twitter handle"></a>
```
[the remark this answers is not in the document]
Comment (SimonGideon) [reply index.html:55]: @christianbiring1  Nice suggestion for the changes. The program instructions, were to modify the portfolio to match as our own.
I implemented the following:
1. The color of the navbar links is animated, the color changes gradually.(From Animations Milestone).

File: index.html [hunk 2]
```diff
@@ -287,13 +286,16 @@ <h2>
...
                   type="text" id="name" name="name" placeholder="user name" maxlength="30" required/>
               </li>
               <li>
+                <label>email-address</label>
                 <input type="email" id="email" name="email" placeholder="example@email.com" required/>
               </li>
               <li>
+                <label>message</label>
                 <textarea type="text" id="name" name="name" placeholder="write your message here" maxlength="500" required ></textarea>
               </li>
```
[the remark this answers is not in the document]
Comment (SimonGideon) [reply index.html:300]: Thanks for this small noted change. The suggested option has been implemented on the latest commit.

File: index.html [hunk 3]
```diff
@@ -219,16 +219,16 @@ <h2>
             <div>
```
[the remark this answers is not in the document]
Comment (SimonGideon) [reply index.html:219]: :partying_face: Didn't see this small color varraition. The above noted changes has been effected on the code to match the Figma design

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (SimonGideon) [reply index.html:55]: All suggestion has been implemented on the code by the latest commit

**Review by ichala — APPROVED** [review]
Hi @SimonGideon ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes implemented ✔️ 

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**